packaging/installer/INSTALL-DEBUG.md

```diff
@@ -0,0 +1,90 @@
+# Troubleshoot problems during Agent installation and known issues
+
+#### Older distributions (Ubuntu 14.04, Debian 8, CentOS 6) and OpenSSL
+
+If you're running an older Linux distribution or one that has reached EOL, such as Ubuntu 14.04 LTS, Debian 8, or CentOS
+6, your Agent may not be able to securely connect to Netdata Cloud due to an outdated version of OpenSSL. These old
+versions of OpenSSL cannot perform [hostname validation](https://wiki.openssl.org/index.php/Hostname_validation), which
+helps securely encrypt SSL connections.
+
+If you choose to continue using the outdated version of OpenSSL, your node will still connect to Netdata Cloud, albeit
+with hostname verification disabled. Without verification, your Netdata Cloud connection could be vulnerable to
+man-in-the-middle attacks.
+
+#### Issues with yum package manager.
+
+In most of the RPM distros that use yum package manager, Our kickstart script is trying to set up our private repo
+to ship the packages and install the native packages of netdata. Open source package maintainers also ship Netdata Agent
+on their distro repos. To instruct the package manager to fetch packages from our repos, we relly upon a concept called
+priorities. The yum package manager needs the `yum-plugin-priorities` package for the package manager to start
+respecting priority rules. Consider installing this plugin, before run the kickstart script.
+
+```sh
+sudo yum install yum-plugin-priorities
+```
+
+#### CentOS 6 and CentOS 8
+
+To install the Agent on certain CentOS and RHEL systems, you must enable non-default repositories, such as EPEL or
+PowerTools, to gather hard dependencies. See
+the [CentOS 6](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/manual.md#centos--rhel-6x) and
+[CentOS 8](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/manual.md#centos--rhel-8x)
+sections for more information.
+
+#### Access to file is not permitted
+
+If you see an error similar to `Access to file is not permitted: /usr/share/netdata/web//index.html` when you try to
+visit the Agent dashboard at `http://NODE:19999`, you need to update Netdata's permissions to match those of your
+system.
+
+Run `ls -la /usr/share/netdata/web/index.html` to find the file's permissions. You may need to change this path based on
+the error you're seeing in your browser. In the below example, the file is owned by the user `root` and the group
+`root`.
+
+```bash
+ls -la /usr/share/netdata/web/index.html
+-rw-r--r--. 1 root root 89377 May  5 06:30 /usr/share/netdata/web/index.html
+```
+
+These files need to have the same user and group used to install your netdata. Suppose you installed netdata with user
+`netdata` and group `netdata`, in this scenario you will need to run the following command to fix the error:
+
+```bash
+# chown -R netdata.netdata /usr/share/netdata/web
+```
+
+
+## Unknown install-type (F050C)
```

Comment:
You can drop the error code here, it should not be user visible from the kickstart script so it’s not useful for users when searching.

If we _really_ want it for some reason, it should be an invisible anchor.

Reply:
Why we dont print this to users? ok sometimes it is a vague code but we can make use of it. Would you like for me to also print this during fatals?

Reply:
Because it’s potentially confusing to users and adds little value for them. The whole reason these codes exist at all is so that we can correlate telemetry events to specific code paths in the kickstart script since we can’t use the error messages themselves for that (they may change over time, some are templated based on information present only at runtime, and some are reused in multiple places).